starter/js/atm.js

```diff
@@ -1,43 +1,29 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
-
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
-
-        //Take that value and add it to the existing value in the checkingBalance div
-
-    //Checking account withdrawl funtion
-
-      //On click of the withdrawChecking button
-
-        //Get value from the amountChecking input field
 
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
-
-        //Else subtract that value from the current amount in the checking account
-
-    //Savings account deposit function
-
-      //On click of the depositSavings button
-
-        //Get value from the amountSavings input field
-
-        //Take that value and add it to the existing value in the savingsBalance div
-
-    //Savings account withdraw funtion
+//Begin with the document ready function
 
-      //On click of the withdrawl button
+$(document).ready (function() {
+    $('#depositChecking').click(function(event) {
+      var depositAmount= parseInt($('#amountChecking').val(), 10);
+      var currentBalanceChecking = parseInt($('#checkingBalance').text().replace('$',''), 10);
+      var newBalance =   depositAmount + currentBalanceChecking;
+      $('#checkingBalance').text("$" + newBalance);
+  });
 
-        //Get value from the amountSavings input field
+      console.log( "ready!");
+    $('#withdrawChecking').click(function(event) {
+      var withdrawAmount= parseInt($('#amountChecking').val(), 10);
+      var currentBalanceChecking = parseInt($('#checkingBalance').text().replace('$',''), 10);
+      var newBalance =   currentBalanceChecking - withdrawAmount;
+      $('#checkingBalance').text("$" + newBalance);
 
-         //If that value is greater than the value in the account ignore that action
-         //In other words if this would put the account into a negative balance do not allow it
+      if (withdrawAmount<=currentBalanceChecking) {
+        $('checkingBalance').text("$" + newBalance);
+      }
+        else {
```

Comment:
The else part of this conditional should have the same indentation. It should look like:

```js
if (withdrawAmoun<=currentBalanceChecking) {
   // code here
} else {
   // more code here
}
```